examples/src/main/java/com/datastax/oss/driver/examples/json/jackson/JacksonJsonColumn.java

```diff
@@ -0,0 +1,171 @@
+import com.datastax.oss.driver.examples.json.codecs.JacksonJsonCodec;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Illustrates how to map a single table column of type {@code VARCHAR}, containing JSON payloads,
+ * into a Java object using the <a href="http://wiki.fasterxml.com/JacksonHome">Jackson</a> library.
+ *
+ * <p>This example makes usage of a custom {@link TypeCodec codec}, {@link JacksonJsonCodec}, which
+ * is implemented in the java-driver-examples module. If you plan to follow this example, make sure
+ * to include the following Maven dependencies in your project:
+ *
+ * <pre>{@code
+ * <dependency>
+ *   <groupId>com.fasterxml.jackson.core</groupId>
+ *   <artifactId>jackson-databind</artifactId>
+ *   <version>2.9.8</version>
+ * </dependency>
+ * }</pre>
+ *
+ * <p>This example also uses the {@link com.datastax.oss.driver.api.querybuilder.QueryBuilder
+ * QueryBuilder}; for examples using the "core" API, see {@link PlainTextJson} (they are easily
+ * translatable to the queries in this class).
+ *
+ * <p>Preconditions:
+ *
+ * <ul>
+ *   <li>An Apache Cassandra(R) cluster is running and accessible through the contacts points
+ *       identified by basic.contact-points (see application.conf).
+ * </ul>
+ *
+ * <p>Side effects:
+ *
+ * <ul>
+ *   <li>creates a new keyspace "examples" in the cluster. If a keyspace with this name already
+ *       exists, it will be reused;
+ *   <li>creates a table "examples.json_jackson_column". If it already exists, it will be reused;
+ *   <li>inserts data in the table.
+ * </ul>
+ */
+public class JacksonJsonColumn {
+
+  // A codec to convert JSON payloads into User instances;
+  private static final TypeCodec<User> USER_CODEC = new JacksonJsonCodec<>(User.class);
+
+  public static void main(String[] args) {
+    try (CqlSession session = new CqlSessionBuilder().addTypeCodecs(USER_CODEC).build()) {
+      createSchema(session);
+      insertJsonColumn(session);
+      selectJsonColumn(session);
+    }
+  }
+
+  private static void createSchema(CqlSession session) {
+    session.execute(
+        "CREATE KEYSPACE IF NOT EXISTS examples "
+            + "WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 1}");
+    session.execute(
+        "CREATE TABLE IF NOT EXISTS examples.json_jackson_column("
+            + "id int PRIMARY KEY, json text)");
+  }
+
+  // Mapping a User instance to a table column
+  private static void insertJsonColumn(CqlSession session) {
+
+    User alice = new User("alice", 30);
+    User bob = new User("bob", 35);
+
+    // Build and execute a simple statement
+
+    Statement stmt =
+        insertInto("examples", "json_jackson_column")
+            .value("id", literal(1))
+            // the User object will be converted into a String and persisted into the VARCHAR column
+            // "json"
+            .value("json", literal(alice, USER_CODEC))
```

Comment:
I think it would make more sense to pass the session's codec registry here, that's simpler when you have an existing session (and not necessarily a reference to the codecs that were registered at init time):
```suggestion
            .value("json", literal(alice, session.getContext().getCodecRegistry()))
```